app/api/session.py

```
@@ -165,10 +155,10 @@ def get_active_thread(session_id: int, db: Session) -> ActiveThreadInfo | None:
 
 @router.get("/current/")
 @limiter.limit("200/minute")
-def get_current_session(
+async def get_current_session(
     request: Request,
```

Ruff (0.14.14)

[warning] Unused function argument: `request`

(ARG001)